### ISSUE TITLE
Containers with multiple links

### ISSUE THREAD

**Issue by samcro1967** [DVDAndroid/glance-docker-container-ext#5]
I have a handful of containers that have more than one url link to access services with no links between them and some are on different ports.  Is there any way currently to have more than one set of labels for a container to create 2 entries for it in the widget?

**Comment by DVDAndroid** [comment]
So, you basically want to create 2+ widgets for the same container, but different URL (or name too); that's correct?

Currently is not possibile, but it may actually be useful. I'll look into it

**Comment by samcro1967** [comment]
I am thinking something along these lines if using docker labels:

**Example docker compose config:**
 ```
      - glance_0.enable=true
      - glance_0.name=AppName - User Interface
      - glance_0.group=UserApps
      - glance_0.url=http://AppName.lan/login
      - glance_1.enable=true
      - glance_1.name=AppName - Admin Interface
      - glance_1.group=AdminApps
      - glance_1.url=https://AppName.lan/admin
```

Another option could be to use json files in a locally mounted folder that is periodically scanned and added and keep only one set of labels in docker compose.  This is how AutoKuma does it for Uptime-Kuma.  Balances keeping what has to be entered and then parsed in docker compose keeping it as it.  Gives users the ability to add more things and still in an automated way.  It is another place to go to create and populate files manually, but this is the exception, not the norm so IMHO it is manageable from a user perspective.

**AppName.json**
```
{
      "glance.enable": "true"
      "glance.containername": "AppName"
      "glance.name": "AppName - Admin Interface"
      "glance.group": "AdminApps"
      "glance.url": "https://AppName.lan/admin"
}
```

Probably other (and better ways to do this), but those are the first two that come to mind based on how I have seen other projects do it.  Appreciate your consideration into adding this capability.

**Comment by DVDAndroid** [comment]
> 
> ```
>      - glance_0.enable=true
>      - glance_0.name=AppName - User Interface
>      - glance_0.group=UserApps
>      - glance_0.url=http://AppName.lan/login
>      - glance_1.enable=true
>      - glance_1.name=AppName - Admin Interface
>      - glance_1.group=AdminApps
>      - glance_1.url=https://AppName.lan/admin
> ```

I was thinking this configuration too, but only with a single `glance.enable` label

**Comment by DVDAndroid** [comment]
Hi, I have published version `2.0.0-beta1` with this new feature. You need to replace all labels with this new format `glance.X.<sublabel>`, where X is a number.

I'll update docs, please give me feedback

**Comment by samcro1967** [comment]
Seeing some weird behavior, but not getting any errors in the logs.  Just to confirm, does every existing label need to change from `glance.<sublabel>` to `glance.X.<sublabel>`?  Or do I only need to do that for a container that I want to have more than one entry for?

And for containers with more than one entry does the first one need to still follow `glance.<sublabel>`?

**Comment by DVDAndroid** [comment]
Every container must follow `glance.X.<sublabel>`. For pre-existing containers you can use this regex `glance.(enable|name|url|group|icon|description)` replaced with `glance.0.$1`.

**Comment by samcro1967** [comment]
Seems like whatsupdocker is taking issue with the image tag `2.0.0-beta1` and is changing the tag in my docker compose file when it scans for new image updates.  I have told it to ignore this image/container and everything seems to be working fine now.

**Before whatsupdocker scans:**
```
  glance-docker-container-ext:
    hostname: glance-docker-container-ext
    image: dvdandroid/glance-docker-container-ext:2.0.0-beta1
    container_name: glance-docker-container-ext
```

**After whatsupdocker scans:**
```
  glance-docker-container-ext:
    hostname: glance-docker-container-ext
    image: dvdandroid/glance-docker-container-ext:7af5def 
    container_name: glance-docker-container-ext
```

**Comment by DVDAndroid** [comment]
Commit hash `7af5def` is the stable version; ignore whatsupdocker for this repo for now

**Comment by DVDAndroid** [comment]
@samcro1967 can you give me feedback about version `2.0.0-beta1` with the new labels changes? Thanks

**Comment by samcro1967** [comment]
Everything has been running without issue the past 48 hours.  I would consider 2.0.0-beta1 as stable at this point.  Only thing you might consider adding is a message to the container logs about the breaking change requiring all labels must now follow the new `glance.X.<sublabel>` standard for those that never read the change logs.

**Comment by DVDAndroid** [comment]
Released `2.0.0-beta2` with some code cleanup and added the deprecation warning. I'll wait 2 days and I'll mark it stable if everything works fine (feedback appreciated)

**Comment by samcro1967** [comment]
Just pulled `2.0.0-beta2` and am now running it.  Seeing no issues and everything is working as expected.
